apps/prairielearn/src/ee/pages/instructorAiGenerateJob/instructorAiGenerateJob.html.ts

```diff
@@ -0,0 +1,55 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobPage({
```

Comment:
As previously requested, please drop the "Page" suffix from the name. You should also ensure that the function name matches the directory name, so in this case `InstructorAiGenerateJob`.

Reply:
Ahh that's what you meant by "don't incline", sorry.

Reply:
I spelled "include" wrong, apologies.

Reply:
Done in deabe2646069d6cce9d383460ea361768ab6c66c